fonts.css

```diff
@@ -0,0 +1,23 @@
+@font-face {
+    font-family: 'Montserrat';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/LOCO__Oxana/pullRequest?prid=5556136)